scripts/reana

```diff
@@ -104,6 +106,134 @@ down () {
     kubectl delete -Rf configuration-manifests
 }
 
+helloworld () {
+
+    # Get URL of reana-workflow-controller
+    controller_url=$(reana get reana-workflow-monitor)
+
+    # Submit a -workflow job to workflow-controller
+    workflow_id=$(curl --fail --silent \
+                    -H "Content-Type: application/json" \
+                    -X POST -d \
+                    "{\"experiment\":\"$experiment\",
+                      \"toplevel\":\"github:diegodelemos/reana-demo-helloworld@2-add-helloworld-workflow\",
+                      \"workflow\": \"hello.yaml\",
+                      \"nparallel\": \"10\",
+                      \"preset_pars\": {\"name\": \"J.Doe\", \"delay\": \"2\"}
+                    }" \
+                    http://"$(reana get reana-workflow-controller)"/yadage \
+                    | grep -oP '(?<="workflow_id": ")[^"]*')
+
+    # Check for successful execution of curl
+    # https://unix.stackexchange.com/a/204790
+    # Note that checking with '$?' might fail on some corner cases
+    # AND when authentication is involved (response codes 401 and 407)
+    submit_status=$?
+
+    if [ "$submit_status" = "0" ]; then
+
+        if [ "$flag_verbose" != "" ]; then
+            echo "[INFO] Open http://$controller_url/$workflow_id to see visual representation of your workflow and it's progress."
+            echo "[INFO] You can 'cat' or 'tail' /reana/$experiment/$workflow_id/yadage/helloworld/greetings.txt -file"
+            echo "       from shared storage of REANA cluster to see how helloworld workflow progresses."
+            echo "[INFO] (e.g. minikube ssh \"tail -c+1 -F /reana/$experiment/$workflow_id/yadage/helloworld/greetings.txt\")"
+        fi
+
+        echo ""
+        echo "Workflow_id: $workflow_id"
+        echo "Monitor: http://$controller_url/$workflow_id"
+        echo "Output:  /reana/$experiment/$workflow_id/yadage/helloworld/greetings.txt"
+        echo ""
+
+        if [ "$flag_devel" != "" ]; then
```

Comment:
We could leave it there if we find it useful for development, all in all it is an option..